src/tools/default_flags_table/reco_flags.py

```diff
@@ -74,7 +74,100 @@
     ('BEMC:EcalBarrelTruthClusters:moduleDimZName',              '',                               ''),
     ('BEMC:EcalBarrelTruthClusters:samplingFraction',            '1',                              '*'),
 
-    # EEMC - Endcap EMC
+    # BEMC - Imaging Barrel
+    #------------------
+    ('BEMC:EcalBarrelImagingRawHits:input_tag',                 'EcalBarrelHits',                  'Name of input collection to use'),
```

Comment:
Yes, we should put this in for SciGlass as well. The parameter actually already exists and is set to the right value in the code. It was just promoted into a config. parameter after this file was created so had not been copied in. Note that setting it here to the same value that is already in the code won't change anything in how the code functions. You're right though that it would make this file more complete. I'll commit the change.